runtime/doc/usr_41.txt

```diff
@@ -1881,7 +1897,7 @@ mapping.  For these commands, any character until the end-of-line or '|' is
 included.  As a consequence of this behavior, you don't always see that
 trailing whitespace is included: >
 
-	map <F4> o#include  
+	map <F4> o#include
```

Comment:
The trailing whitespace here is required.

Reply:
Auch! I have an autocommand that trim the whitespaces on save.